TMTumblrSDK/Core/TMSDKFunctions.h

```diff
@@ -6,6 +6,7 @@
 //  Copyright (c) 2013 Tumblr. All rights reserved.
 //
 
+@import Foundation;
```

Comment:
A lot of these @imports duplicate the other imports, and will @import break when this is compiled into a static lib?